tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -131,7 +131,18 @@ public:
    /// A bare model has no default entry
    static std::unique_ptr<RNTupleModel> CreateBare();
 
-   /// Creates a new field and a corresponding tree value that is managed by a shared pointer.
+   struct NameWithDescription_t {
+      NameWithDescription_t(const char* name): fName(name) {}
```

Comment:
I think we can remove this, as a `std::string_view` can be implicitly constructed from `const char *`.

Reply:
Argh bummer I forgot your hint on that, sorry Javier! I will hand in a follow-up PR.

Reply:
That gives
```
tree/ntuple/v7/test/ntuple_cluster.cxx: In member function ‘virtual void PageStorageFile_LoadClusters_Test::TestBody()’:
tree/ntuple/v7/test/ntuple_cluster.cxx:289:44: error: no matching function for call to ‘ROOT::Experimental::RNTupleModel::MakeField<float>(const char [3], double)’
  289 |    auto wrPt = modelWrite->MakeField<float>("pt", 42.0);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
In file included from tree/ntuple/v7/inc/ROOT/RNTuple.hxx:23,
                 from tree/ntuple/v7/test/ntuple_cluster.cxx:7:
tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx:189:23: note: candidate: ‘template<class T, class ... ArgsT> std::shared_ptr<_Tp> ROOT::Experimental::RNTupleModel::MakeField(const NameWithDescription_t&, ArgsT&& ...)’
  189 |    std::shared_ptr<T> MakeField(const NameWithDescription_t &fieldNameDesc,
      |                       ^~~~~~~~~
tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx:189:23: note:   template argument deduction/substitution failed:
tree/ntuple/v7/test/ntuple_cluster.cxx:289:45: note:   cannot convert ‘"pt"’ (type ‘const char [3]’) to type ‘const ROOT::Experimental::RNTupleModel::NameWithDescription_t&’
  289 |    auto wrPt = modelWrite->MakeField<float>("pt", 42.0);
      |                                             ^~~~
```
Maybe because *two* implicit conversions (string literal to `string_view`, `string_view` to `NameWithDescription_t`) aren't taken into account for finding candidates?

I won't open a PR :-)

Reply:
That's right! :+1:  Fair enough :-).